client/app/screens/InviteList.js

```diff
@@ -83,7 +84,16 @@ export default class InviteListScreen extends React.Component {
           });
       });
 
-    // Initial load of invites
+    let reloadInvites = function(invites) {
+      this.setState({ dataSource: ds.cloneWithRows(invites) });
+    }.bind(this);
+
+    DeviceEventEmitter.addListener('onDefaultMessage', function(event) {
```

Comment:
It looks like this method of sending events from Java to JS is deprecated and no longer documented — see https://github.com/facebook/react-native/pull/12867/commits/b11f8f4936459c104b4fccb3da5eef1d53aa76f7. Not sure what the alternative is though...